samples/Mvc.LocalizationSample.Web/Models/Product.cs

```diff
@@ -7,12 +7,15 @@ namespace Mvc.LocalizationSample.Web.Models
 {
     public class Product
     {
-        [Required(ErrorMessage = "ProductName")]
+        [Display(Name = "Name")]
```

Comment:
😢 I liked "Product Name". More importantly this and a `User`'s name now have the same label in the generated HTML. Suggest going back to "Product Name" here to remove that ambiguity.